pkgs/applications/altcoins/bitcoin.nix

```diff
@@ -6,13 +6,14 @@ with stdenv.lib;
 stdenv.mkDerivation rec{
 
   name = "bitcoin" + (toString (optional (!withGui) "d")) + "-" + version;
-  version = "0.10.0";
+  core_version = "0.10.1";
```

Comment:
There may be none. I just saw the url, saw there were two numeric sections in it, and concluded they were distinct for a reason.